### PR TITLE
support macOS

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Review by pawosm-arm — APPROVED** [review]

File: clang/lib/Driver/ToolChains/Darwin.cpp [hunk 1]
```diff
@@ -614,6 +614,16 @@ void darwin::Linker::ConstructJob(Compilation &C, const JobAction &JA,
   if (getToolChain().ShouldLinkCXXStdlib(Args))
     getToolChain().AddCXXStdlibLibArgs(Args, CmdArgs);
 
+  // Add Fortran runtime libraries
+  if (needFortranLibs(getToolChain().getDriver(), Args)) {
+    getToolChain().AddFortranStdlibLibArgs(Args, CmdArgs);
+  } else {
+  // Claim "no Flang libraries" arguments if any
```
Comment (bryanpkc) [inline clang/lib/Driver/ToolChains/Darwin.cpp]: Please indent this line.

**Review by bryanpkc — APPROVED** [review]
LGTM except for a nit.

File: clang/lib/Driver/ToolChains/Darwin.cpp (hunk 1 above)
Comment (isuruf) [reply clang/lib/Driver/ToolChains/Darwin.cpp]: Done in 608e3f8

**Comment by bryanpkc** [comment]
I missed the fact that the branch had multiple commits, and should have squashed the branch before merging. 😢  

@isuruf Is this needed in the release_11x and release_12x branches?

**Comment by isuruf** [comment]
> I missed the fact that the branch had multiple commits, and should have squashed the branch before merging. :cry:

Can you squash the branch?

> @isuruf Is this needed in the release_11x and release_12x branches?

Yes, shall I send PRs for these branches?

**Pull request merged**